scripts/target.rb

```diff
@@ -8,7 +8,8 @@ class Target
       'openstack' => ImagefactoryMetadata.new('openstack-kvm', nil, 'qc2'),
       'hyperv'    => ImagefactoryMetadata.new('hyperv', nil, 'vhd'),
       'azure'     => ImagefactoryMetadata.new('hyperv', nil, 'vhd'),
-      'vagrant'   => ImagefactoryMetadata.new('vsphere', 'vagrant-virtualbox', 'box')
+      'vagrant'   => ImagefactoryMetadata.new('vsphere', 'vagrant-virtualbox', 'box'),
+      'gce'       => ImagefactoryMetadata.new('gce', 'tar.gz')
```

Comment:
With the new changes from @geertj This line needs 3 parameters now.  This *should* be blowing up if you run through it.